test/Swashbuckle.AspNetCore.IntegrationTests/Swashbuckle.AspNetCore.IntegrationTests.csproj

```diff
@@ -42,6 +42,15 @@
     <PackageReference Include="xunit.v3" />
   </ItemGroup>
 
+  <ItemGroup>
+    <None Include="../../src/Swashbuckle.AspNetCore.SwaggerUI/node_modules/swagger-ui-dist/**/*" Exclude="**/*/index.html;**/*/index.js;**/*/*.map;**/*/*.json;**/*/*.md;**/*/swagger-ui-es-*;**/*/LICENSE;**/*/NOTICE" Link="swagger-ui-dist/%(RecursiveDir)%(FileName)%(Extension)">
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+    </None>
+    <None Include="../../src/Swashbuckle.AspNetCore.ReDoc/node_modules/redoc/bundles/redoc.standalone.js" Link="redoc/bundles/redoc.standalone.js">
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+    </None>
+  </ItemGroup>
```

Comment:
Rather than copy all these files around, I would suggest instead doing something like this:

1. [Embed the full path of the solution into the test assembly](https://github.com/martincostello/build-kit/blob/5e1fbfd37b76d040c7b9c5a99ecaaac1ea43ae03/tests/BuildKit.Tests/MartinCostello.BuildKit.Tests.csproj#L33)
2. [Extract the value from the assembly at runtime in the tests to find the original file on disk directly](https://github.com/martincostello/build-kit/blob/5e1fbfd37b76d040c7b9c5a99ecaaac1ea43ae03/tests/BuildKit.Tests/IntegrationTests.cs#L12-L16)